packages/components/src/styles/select/__snapshots__/option-styles.spec.ts.snap

```diff
@@ -0,0 +1,201 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getSelectOptionStyles() should return correct css for theme: dark and additionalOptionJssStyle: { color: 'black' } 1`] = `
```

Comment:
The returned and snapshotted results are not css but more like a jss `Styles` object.